be/src/formats/parquet/group_reader.cpp

```diff
@@ -385,7 +383,7 @@ void GroupReader::_init_read_chunk() {
     const auto& slots = _param.tuple_desc->slots();
     std::vector<SlotDescriptor*> read_slots;
     for (const auto& column : _param.read_cols) {
-        int chunk_index = column.col_idx_in_chunk;
+        int chunk_index = column.idx_in_chunk;
         read_slots.emplace_back(slots[chunk_index]);
```

Comment:
I think this is the final reason, idx_in_chunk use both tuple_desc->slots() and delete_column_tuple_desc->slots(), which lead to oversize.

Reply:
yes, see this pr to fix https://github.com/StarRocks/starrocks/pull/42087

Reply:
so we'd better not to use `idx_in_chunk`, but to use slot desc, not matter it's from tuple desc or delete column tuple desc.